lib/support/cli.js

```diff
@@ -283,7 +283,6 @@ module.exports.parseCommandLine = function parseCommandLine() {
     })
     .option('firefox.geckoProfilerParams.interval', {
       describe: 'Sampling interval in ms.',
```

Comment:
Consider including conditional default in the description, although that can itself get out of date.

Reply:
> I'm not sure if we can be certain the target platform is android at this point yet.

Right -- I mean, `describe: 'Sampling interval in ms; defaults to 1ms on Desktop and 4ms on Android.'`.